src/frontend-samples/app-ui-samples/viewport-frontstage-sample/ViewportFrontstageApp.tsx

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import "../../../common/samples-common.scss";
+import { SampleAppUiComponent } from "../../../common/AppUi/SampleAppUiComponent";
+import { AppUi } from "../../../common/AppUi/AppUi";
+
+// The Props and State for this sample component
+/** A React component that renders the UI specific for this sample */
+export class ViewportFrontstageSample extends React.Component < { iModelSelector: React.ReactNode } > {
+  public static async setup(iModelName: string, iModelSelector: React.ReactNode) {
+    // Initialize utility class for AppUi samples
+    AppUi.initialize();
+    // set up iModel and AppUi Frontstage
+    await AppUi.setIModelAndFrontstage(iModelName, "ViewportFrontstage");
+    return <ViewportFrontstageSample iModelSelector={ iModelSelector }></ViewportFrontstageSample>;
+  }
+  public static teardown() {
+    AppUi.restoreDefaults();
+  }
+
+  /** The sample's render method */
+  public render() {
+    return (
+      <>
+        <SampleAppUiComponent></SampleAppUiComponent>
+        <div className="sample-ui">

```

Comment:
This covers up the SmallStatusBar
![image](https://user-images.githubusercontent.com/38542573/88576834-0b8dff00-d014-11ea-92cd-c34c846658c1.png)

Reply:
Should I move it for the AppUI samples only?

Reply:
I think so. It is in good place for the samples that do not have the SmallStatusBar.

Reply:
I added a class to change the location of the bottom of the instructions window for AppUI samples specifically.